apps/condo/domains/meter/schema/Meter.test.js

```diff
@@ -793,10 +804,21 @@ describe('Meter', () => {
                 unitName,
             })
 
-            await updateTestProperty(adminClient, property.id, { deletedAt: 'true' })
+            await PropertyAPI.softDelete(adminClient, property.id)
+
+            // NOTE: give worker some time
+            await sleep(1000)
```

Comment:
Yes. The real work is on worker side. You can check it only by db state changes or by sleep. 

If the testers did it, he would do it as well.

Reply:
In CI, tests are run without fake mode. A full-featured web server with a worker is being up and running.

And the test client use a http client. 

So that the tests are as close as possible to the real life of the tester.

Reply:
https://doma.atlassian.net/browse/DOMA-1754
Will follow it up later.